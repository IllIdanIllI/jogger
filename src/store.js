import { createStore, combineReducers, applyMiddleware } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import authentication from './reducers/authenticationReducer';
import jogStore from './reducers/jogsReducer';
const rootReducer = combineReducers({
    authentication,
    jogStore,
});

const middleware = [thunk];

const store = createStore(
    rootReducer,
    composeWithDevTools(applyMiddleware(...middleware))
);
export default store;
