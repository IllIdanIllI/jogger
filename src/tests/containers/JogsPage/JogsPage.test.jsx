import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../../store';
import JogPage from '../../../containers/JogsPage';
import renderer from 'react-test-renderer';

test('Jog page test', () => {
    const component = renderer.create(
        <Provider store={store}>
            <Router>
                <JogPage />
            </Router>
        </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
