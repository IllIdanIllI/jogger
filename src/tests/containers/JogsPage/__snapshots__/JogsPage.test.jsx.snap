// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jog page test 1`] = `
<div
  className="empty-jogs-container"
>
  <div
    className="empty-jogs-container__notification"
  >
    <svg
      className="empty-jogs-container__notification-emoji"
    >
      empty-page-emoji.svg
    </svg>
    <span
      className="empty-jogs-container__notification-text"
    >
      Nothing is there
    </span>
  </div>
  <a
    href="/jogs/add"
    onClick={[Function]}
  >
    <button
      className="empty-jogs-container__creation"
    >
      <span>
        Create your first jog
      </span>
    </button>
  </a>
</div>
`;
